### DISCUSSION TITLE
2009/12/29/labview-2009-%E6%96%B0%E5%8A%9F%E8%83%BD-%E5%9B%BE%E6%A0%87%E7%BC%96%E8%BE%91%E5%99%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#287] [Announcements]
# 2009/12/29/labview-2009-%E6%96%B0%E5%8A%9F%E8%83%BD-%E5%9B%BE%E6%A0%87%E7%BC%96%E8%BE%91%E5%99%A8

与文本语言相比，图形化的 LabVIEW VI 有个非常好的特点，就是使用图标来表示 VI 的功能。这样，在程序中，看一眼子VI的图标，就大致知道它的作用了。这比看枯燥的函数名方便迅速多了。 如下图这个VI，一看就知道它和生成波形有关。

https://blog.qizhen.xyz/2009/12/29/labview-2009-%E6%96%B0%E5%8A%9F%E8%83%BD-%E5%9B%BE%E6%A0%87%E7%BC%96%E8%BE%91%E5%99%A8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

志恒
2010年01月22日 20:16 编辑
新版图标编辑器的图标全部以png格式存在我的文档的“LabVIEW Data”中，使用新版图标编辑器的时候需要调用，一旦隐藏后调用失败就会启动旧版。

回复
志恒
2010年01月22日 20:10 编辑
我发现了一个更简单的禁用方法。把我的文档下的“LabVIEW Data”文件夹隐藏后，打开“图标编辑器”以后新版的图标编辑器一闪而过，然后出现旧版。如果要用新版的话取消文件夹隐藏就可以了。

回复
l
2009年12月31日 17:02 编辑
我已经写好了一篇blog并且发送了引用链接,你可以在下面的Trackback看到

回复
Qizhen
2009年12月31日 11:21 编辑
Cleartype在哪里设置，我还真不知道，能详细说一下吗？我用开源这个词可能不太好，严谨的说应该是“源代码公开的” :)

回复
l
2009年12月31日 09:56 编辑
字体模糊是因为打开了Cleartype的缘故,禁用就可以了.另外,开源不只是能查看源代码,还和授权协议有关,这个图标编辑器的授权应该是包含在LabVIEW里面的,是否能算作开源值得商榷.

回复
Qizhen
2009年12月30日 11:01 编辑
禁用新图标编辑器的方法是 Billy Wei 告诉我的，在此表示感谢 :)

回复
